platform/ios/src/MGLCompassButton.mm

```diff
@@ -63,9 +63,17 @@ - (UIImage *)compassImage {
     UIImage *scaleImage = [UIImage mgl_resourceImageNamed:@"Compass"];
     UIGraphicsBeginImageContextWithOptions(scaleImage.size, NO, UIScreen.mainScreen.scale);
     [scaleImage drawInRect:{CGPointZero, scaleImage.size}];
+    
+    UIFont *northFont;
+    if(@available(iOS 13, *)){
+        northFont = [UIFont systemFontOfSize:11 weight:UIFontWeightLight];
+    }
```

Comment:
It’s not possible to suggest multiline edits on GitHub, so this line gets deleted (but is included in another suggested edit).
```suggestion
```